### ISSUE TITLE
Open Browser For Facebook Auth

### ISSUE THREAD

**Issue by kaufmann42** [davicorreiajr/spotify-now-playing#32]
## What are you reporting?

* [ ] Bug, or some problem;
* [x] Improvement in an existing feature;
* [ ] Request for a new feature;
* [ ] A question.

## In which context are you running the app?
Mac OS Mojave V 10.14.2
Now Playing - 0.6.1

## Description
I think it's a little bit sketchy to let a 3rd party application like this authenticate FB credentials within its own web-view. I know you guys aren't doing anything with nefarious, but it is a concern UX wise.

Also it's pretty annoying to have to fetch my credentials on FB / remember my password. Would be great if the app could open up the browser with the OAuth link and then poll the response.

**Comment by paced** [comment]
Agreed, I think this is a pretty important feature. I'd be keen to tackle this once my other three PRs are reviewed if nobody else wants to PR?

**Comment by paced** [comment]
@kaufmann42 I'm wary of hijacking a repo's issues page, however defunct, to discuss my own work, but my hard-fork _Now Playing for Spotify_ has just resolved this in `v0.10.0` which you can [find here](https://github.com/teaminkling/mac-spotify-np/releases/tag/v0.10.0).

There are a number of other changes from `v0.7.0` but I think they are overall worth the switch, if you haven't taken a look.
